ports/ffmpeg/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "ffmpeg",
   "version-string": "4.4",
```

```suggestion
  "version": "4.4",
```
Should be `version` here. Please see the reference https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version.